### DISCUSSION TITLE
External IP

### DISCUSSION THREAD

**Discussion by k2patel** [jesec/flood#171] [Ideas]
Hello,

It would be great to have the functionality to display external IP, it is useful confirming VPN connection and the other aspects.

Thank you.

**Comment by jesec** [comment]
It depends on the torrent client. Flood's external IP may or may not represent the outgoing IP of torrent client. It can be misleading to detect external IP from Flood.

**Reply by ajhorvath20** [reply]
In my case I do `docker exec -it rflood curl icanhazip.com` to get the IP if we can display that IP in Flood UI that would be so handy

![image](https://user-images.githubusercontent.com/65309559/119938298-198a1480-bf84-11eb-91c1-0a077e847a5b.png)
